client/src/App.js

```diff
@@ -138,6 +138,7 @@ toggleBurger = () => {
 
 onSetSidebarOpen(open) {
   this.setState({ sidebarOpen: open });
+  this.toggleBurger()   // reset burger
```

Comment:
One note here. Do you see how we could have consolidated the sidebarOpen and burger state variables into a single boolean value? Not required to merge, but something to consider for the future.

Reply:
Good point. I will try to refactor it when I take another look at it.